src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -144,6 +144,25 @@ void BasePlaylistFeature::initActions() {
             &QAction::triggered,
             this,
             &BasePlaylistFeature::slotExportTrackFiles);
+#ifdef __ENGINEPRIME__
+    // Note: "Engine DJ" is a product name and must not be translated.
```

Comment:
Use `//:` to make a "translator comment". Then the translator can see it in the translation tool.
```suggestion
    //: "Engine DJ" is a product name and must not be translated.
```
same below

Reply:
Thanks, done.